src/styles/bloks/how-to-buy.scss

```diff
@@ -0,0 +1,45 @@
+.how-to-buy {
+  @include pageGrid;
+
+  &__title {
+    margin: 0;
+    grid-column: 1 / -1;
+
+    @include onTablet {
+      grid-column: 1 / 4;
+    }
+
+    @include onDesktop {
+      grid-column: span 5;
+    }
+  }
+
+  &__content {
+    grid-column: span 1;
+
+    @include onTablet {
+      grid-column: span 3;
+    }
+
+    @include onDesktop {
+      grid-column: span 5;
+    }
+  }
+
+  &__benefits {
+    display: grid;
+    row-gap: 16px;
+
+    margin: 0;
+    margin-bottom: 40px;
+    padding: 0;
+    list-style: none;
+    font-weight: bold;
+
+  }
+
+  &__description {
+    margin: 0;
+    line-height: 27px;
```

Comment:
It's better to use relative properties for line height. It could help to change this value accordingly to the text size